### PR TITLE
Wayfinder v2.4

### PR DESCRIPTION
**Pull request by KashifSaadat**
- Module changes to support Wayfinder v2.4 (release v2.4.2)
- EKS updated to v1.26

**Important:**
- Wayfinder's cloud access will be recreated with the switch to re-use the cloudaccess submodule. If you are using the Global DNS Zone feature within Wayfinder but managed by the same AWS Account that Wayfinder runs within, you must pass the variable `enable_wf_dnszonemanager = true` to the module to ensure the relevant role is setup correctly.

### CODE REVIEW

File: external-dns.tf [hunk 1]
```diff
@@ -25,9 +25,9 @@ resource "helm_release" "external-dns" {
   create_namespace = true
 
   name        = "external-dns"
-  repository  = "https://charts.bitnami.com/bitnami"
+  repository  = "https://kubernetes-sigs.github.io/external-dns"
   chart       = "external-dns"
-  version     = "6.18.0"
+  version     = "1.13.1"
```
Comment (mrsheepuk) [inline external-dns.tf:30]: out of interest, does this change work correctly over an already-installed cluster? switching the upstream helm package? or would this change only work on a new instance?

File: modules/cloudaccess/examples/complete/outputs.tf [hunk 2]
```diff
@@ -1,29 +1,15 @@
...
-  value       = module.wayfinder_cloudaccess.network_manager_role_arn
+  value = module.wayfinder_cloudaccess.network_manager_role_arn
 }
 
-output "network_manager_role_name" {
-  description = "The name of the Network Manager IAM Role"
-  value       = module.wayfinder_cloudaccess.network_manager_role_name
+output "cloud_info_role_arn" {
+  value = module.wayfinder_cloudaccess.cloud_info_role_arn
 }
```
Comment (mrsheepuk) [inline modules/cloudaccess/examples/complete/outputs.tf:15]: I don't think the new outputs are correct in the example

File: external-dns.tf (hunk 1 above)
Comment (KashifSaadat) [reply external-dns.tf:30]: It does work yeah on an already-installed cluster, but a one-off in this case and I'd not recommend / risk it in the future. Probably instead switch the release name to ensure it's unique.

**Review by mrsheepuk — APPROVED** [review]
I'm happy for this to go in - all good for v2.4.3.

**Pull request merged**